### PR TITLE
CORE-120 - Non-Production Environment Notification

### PR DESCRIPTION
**Pull request by atoff**
image
image

### CODE REVIEW

**Comment by AxonC** [comment]
Could we have it above the navigation bar? I feel I'd notice that more.

**Comment by CalumTowers** [comment]
Thinking about it, would we be better having this as a separate env variable?

**Comment by CalumTowers** [comment]
You’re right. My thinking is it may become a hindrance for developing (seeing what it will actually look like)

My main aim for this is for alpha/beta testing, rather than local envs. 

I think we can kill two birds with one stone by setting this to only display if the env is staging?

**Comment by atoff** [comment]
I think that sounds good to me. I may also implement a optional .env variable as well so that you can use it on dev as I know I would like it so I can see at a glance I am not on live! Will action tomorrow

File: app/helpers.php [hunk 1]
```diff
@@ -176,3 +176,7 @@ function minutesToHours($minutes)
 
     return sprintf('%02.2d:%02.2d', floor($seconds / 3600), $seconds / 60 % 60);
 }
+
+function is_local_environment() {
+    return env("APP_IS_LOCAL") == "true" || App::environment('staging');
```
Comment (CalumTowers) [inline app/helpers.php]: A staging env isn't really local, we should probably restrict this to just return the env true/false

**Pull request merged**